### PR TITLE
Cis 27 frontend app routing footer

### PR DESCRIPTION
**Pull request by ThatZiro**
# JIRA Ticket:

Please include a link to the JIRA ticket that this pull request is related to:
https://mammoth-studios.atlassian.net/browse/CIS-27

## Description

CIS-27 Built out Footer Component
- Created the Footer Component
- Added Placeholder logo
- Added Placeholder links

Notes : 
- Footer is built using Anchors, should we move it over to React Router Links? *answered*
- Links are currently placeholders to be changes later
- Need a Logo to place on the footer
- Should I remove additional files from the Footer component? *answered*

## Type of Change

- [ ] Bug fix
- [x] New feature
- [ ] Refactor

## Screenshot / Video
mobile (7)
mobile (6)

### CODE REVIEW

File: front-end/src/lib/Footer/Footer.component.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            d="M64 0C28.7 0 0 28.7 0 64V384C0 419.3 28.7 448 64 448H162.2V302.2H109.4V224H162.2V190.3C162.2 103.2 201.6 62.8 287.2 62.8C303.4 62.8 331.4 66 342.9 69.2V140C336.9 139.4 326.4 139 313.3 139C271.3 139 255.1 154.9 255.1 196.2V224H338.7L324.3 302.2H255V448H384C419.3 448 448 419.3 448 384V64C448 28.7 419.3 0 384 0H64Z"
+          />
+        </svg>
+      </a>
+    </div>
+    <p>© 2024 Mammoth Studios, All rights reserved.</p>
+  </footer>
+);
+
+export default Footer;
```
Comment (dustinjack99) [inline front-end/src/lib/Footer/Footer.component.tsx]: If you do a non-default export, you'll be able to do the cool index.ts import in all your files. See above comment 😄

File: front-end/src/App.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { Provider } from "react-redux";
 import { ExampleFeature } from "./domains/example-feature/example-feature.index";
 import { store } from "./redux/store";
+import Footer from "./lib/Footer/Footer.component";
```
Comment (dustinjack99) [inline front-end/src/App.tsx]: If we export Footer.component correctly, you'll only need to write:

`import Footer from "./lib/Footer";` 

This is one of the many benefits of having an index file that exports the composition of all other elements in Footer.component.tsx 💡

File: front-end/src/lib/Footer/Footer.component.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      <a href="/contact" aria-label="Contact Page">Contact</a>
+      <a href="/how-to-play" aria-label="How To Play Page">How To Play</a>
+      <a href="/policy/privacy" aria-label="Privacy Policy Page">Privacy Policy</a>
+      <a href="/policy/TOS" aria-label="Terms Of Service Page">Terms of Service</a>
+      <a href="/careers" aria-label="Careers Page">Careers</a>
+    </div>
+    <div className="socials">
+
+      <a href="https://www.facebook.com" aria-label="www.Facebook.com">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448" fill="none">
```
Comment (dustinjack99) [inline front-end/src/lib/Footer/Footer.component.tsx]: For readability, I don't think it's a bad idea to have each SVG element as its own component.

Like:
- facebook-logo.component.tsx
- x-logo.component.tsx

Then we can use them in other places if needed 😄

File: front-end/src/lib/Footer/Footer.queries.ts
Comment (dustinjack99) [inline front-end/src/lib/Footer/Footer.queries.ts:1]: Feel free to delete these empty files 👍

**Review by dustinjack99 — CHANGES_REQUESTED** [review]
Overall GREAT first PR! 

Love the screenshots and care taken for responsive design - and the logos look fantastic! Great work 💪

File: front-end/src/lib/Footer/Footer.component.tsx (hunk 3 above)
Comment (ThatZiro) [reply front-end/src/lib/Footer/Footer.component.tsx]: That's something I can setup! Where would you like these components to live?

File: front-end/src/App.tsx (hunk 2 above)
Comment (ThatZiro) [reply front-end/src/App.tsx]: I'll make sure that is working correctly thank you for catching that!

**Review by dustinjack99 — APPROVED** [review]
Perfection 🤌 🤌

**Pull request merged**